schema/config-linux.json

```diff
@@ -273,7 +273,11 @@
                     "l3CacheSchema": {
                         "id": "https://opencontainers.org/schema/bundle/linux/intelRdt/l3CacheSchema",
                         "type": "string"
-                    }
+                    },
+                    "memBwSchema": {
+                        "id": "https://opencontainers.org/schema/bundle/linux/intelRdt/memBwSchema",
+                        "type": "string"
```

Comment:
Can we use a regexp for validation (like we do [here][1] and other places)?  That would be:

```json
"memBwSchema": {
  "id": "https://opencontainers.org/schema/bundle/linux/intelRdt/memBwSchema",
  "type": "string",
  "pattern": "^MB:[^\n]*"
}
```

And we can unit-test the schema on this point with an entry in [here][2].

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/schema/defs-linux.json#L121
[2]: https://github.com/opencontainers/runtime-spec/tree/v1.0.0/schema/test/config/bad

Reply:
@wking 

Good idea. I'd like to add this.
We could use "./schema/validate" to validate the format of `memBwSchema` in config.json.